recipes/correct-ts-specifiers/package.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "correct-ts-specifiers",
+  "version": "1.0.0-rc.1",
+  "description": "Replace erroneous 'js' or omitted file extensions of import specifiers in TypeScript files.",
+  "type": "module",
+  "main": "./src/workflow.ts",
+  "engines": {
+    "node": ">=22"
+  },
+  "scripts": {
+    "start": "node --no-warnings --experimental-import-meta-resolve --experimental-strip-types ./src/workflow.ts",
+    "test": "node --no-warnings --experimental-import-meta-resolve --experimental-test-module-mocks --experimental-test-snapshots --experimental-strip-types --test --experimental-test-coverage --test-coverage-include='src/**/*' --test-coverage-exclude='**/*.test.ts' './**/*.test.ts'"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/JakobJingleheimer/correct-ts-specifiers.git"
+  },
+  "keywords": [
+    "codemod",
+    "esm",
+    "typescript"
+  ],
+  "author": "Jacob Smith",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/JakobJingleheimer/correct-ts-specifiers/issues"
+  },
+  "homepage": "https://github.com/JakobJingleheimer/correct-ts-specifiers#readme",
+  "devDependencies": {
+    "@types/lodash.get": "^4.4.9",
+    "@types/node": "^22.3.0",
+    "nodejs-loaders": "^1.0.0",
```

```suggestion
    "nodejs-loaders": "~1.1.0",
```

We should ping loader to keep same behavior over the time

---

Ping loader?

btw, before `correct-ts-specifiers` is published, `nodejs-loaders` will be republished as `@nodejs-loaders/…` (this needs only `@nodejs-loaders/alias`).

---

+ 1 for `@nodejs-loaders/alias`

I always ping my loader because it's part of the codebase that affects the runtime. So I make sure I know the behavior

---

OH! Do you mean "pin", like to a fully explicit version? These are fairly stable, and we own the loader, so I think this should be okay.

---

yeah sorry for the typo.

>  like to a fully explicit version?

Yeah

---

I think for others, maybe yes. I think these are okay.